apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chatlist/ChatListView.kt

```diff
@@ -91,13 +116,20 @@ fun ChatListView(chatModel: ChatModel, settingsState: SettingsViewState, setPerf
     drawerGesturesEnabled = appPlatform.isAndroid,
     floatingActionButton = {
       if (searchText.value.text.isEmpty() && !chatModel.desktopNoUserNoRemote && chatModel.chatRunning.value == true) {
+        var bottom = DEFAULT_PADDING
+        if (oneHandUI.state.value) {
+          bottom = DEFAULT_BOTTOM_PADDING
+        } else {
+          bottom -= 16.dp
+        }
```

Comment:
```suggestion
        val bottomPadding = if (oneHandUI.state.value) DEFAULT_BOTTOM_PADDING else DEFAULT_PADDING - 16.dp
```

Reply:
if you don't mind will skip these as per offline chat with EP. we will both remove the anymation and also stop having this button altogether in one hand